mlpp/data_collection/Testing_new_time_filtering_function.ipynb

```diff
@@ -0,0 +1,1259 @@
+       " 2032376,\n",
+       " 3098455,\n",
+       " 17440795,\n",
+       " 11739460,\n",
+       " 17971710,\n",
+       " 18473727,\n",
+       " 10705095,\n",
+       " 16856060,\n",
+       " 11417914,\n",
+       " 16947179,\n",
+       " 15165485,\n",
+       " 13765431,\n",
+       " 17892407,\n",
+       " 17110606,\n",
+       " 14587587,\n",
+       " 15031820,\n",
+       " 12840351,\n",
+       " 12830524,\n",
+       " 17160531,\n",
+       " 17558338,\n",
+       " 16564563,\n",
+       " 13900378,\n",
+       " 2091291,\n",
+       " 16148590,\n",
+       " 13843180,\n",
+       " 13771987,\n",
+       " 12784669,\n",
+       " 15053397,\n",
+       " 17312413,\n",
+       " 17287645,\n",
+       " 16991968,\n",
+       " 7502641,\n",
+       " 13676901,\n",
+       " 11616397,\n",
+       " 17491170,\n",
+       " 15829151,\n",
+       " 13941757,\n",
+       " 10422635,\n",
+       " 4050223,\n",
+       " 17016634,\n",
+       " 11415992,\n",
+       " 13739137,\n",
+       " 10679383,\n",
+       " 13607558,\n",
+       " 1471593,\n",
+       " 13209144,\n",
+       " 4217089,\n",
+       " 16801488,\n",
+       " 11666430,\n",
+       " 15930572,\n",
+       " 8396713,\n",
+       " 7289462,\n",
+       " 16155788,\n",
+       " 16432654,\n",
+       " 12365378,\n",
+       " 17374954,\n",
+       " 13711055,\n",
+       " 17542520,\n",
+       " 15933932,\n",
+       " 18231055,\n",
+       " 17190223,\n",
+       " 14613735,\n",
+       " 15446378,\n",
+       " 15233529,\n",
+       " 18836281,\n",
+       " 12602587,\n",
+       " 15463813,\n",
+       " 8494007,\n",
+       " 12750762,\n",
+       " 13963716,\n",
+       " 15442445,\n",
+       " 5359771,\n",
+       " 8693126,\n",
+       " 14650006,\n",
+       " 8693736,\n",
+       " 18935177,\n",
+       " 13421430,\n",
+       " 14588694,\n",
+       " 14763220,\n",
+       " ...]"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "test.use_random_sample(\"sample_scores_1M\", 6500)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 83,
+   "id": "moderate-chamber",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# data1 = mongo_inst[\"osu_random_db\"]\n",
+    "# test1 = osuDumpSampler(data1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "forbidden-eating",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "collection = data[\"sample_scores_1M\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "plastic-sigma",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test.get_more_recent_than(collection, 2018, 12, 1, 0, 0, 0, \"sample_scores_500k\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "polish-austin",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# test1.use_random_sample(\"sample_scores_100\", 100)"
```

Comment:
same as https://github.com/the-data-science-union/dsu-mlpp/pull/19/files#r568775705